### PR TITLE
Floating labels: Add an artificial background to label

### LINKED ISSUE #32800: Floating label on textarea overlaps content
- Operating system and version: macOS 11.1
- Browser and version: Chrome 87.0.4280.141 (Official Build) (x86_64)

Bug:
image

To reproduce:
1. Visit https://getbootstrap.com/docs/5.0/forms/floating-labels/#textareas
2. Type in 4 lines of text

### PR DESCRIPTION
**Pull request by louismaximepiton**
Fixes #32800.

Here is a proposal for floating labels in textareas.

### Solution

Patchwork of many comments/solution wrote in #32800. I tried to not add an extra class but it might be harder to maintain, I let you decide what would be better here.

### Checks

- Works with everything in the page (even valid and invalid state, select)
- Works with sizes on input
- Couldn't spot any breaking change with the actual implementation

### Live preview

https://deploy-preview-37125--twbs-bootstrap.netlify.app/docs/5.2/forms/floating-labels/#textareas

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -1001,6 +1001,7 @@ $form-floating-padding-x:         $input-padding-x !default;
 $form-floating-padding-y:         1rem !default;
 $form-floating-input-padding-t:   1.625rem !default;
 $form-floating-input-padding-b:   .625rem !default;
+$form-floating-label-height:      1.875em !default;
```
Comment (mdo) [inline scss/_variables.scss:1004]: Maybe we should use a calc function or some math here to get the `1.875em`?

Comment (louismaximepiton) [reply scss/_variables.scss:1004]: Actually, I've chosen this value arbitrarily, it _"feels right" to me. Open to any thoughts here!

**Review by mdo — APPROVED** [review]

**Review by julien-deramond — APPROVED** [review]
LGTM as well! 🚀 
Thanks @louismaximepiton

**Comment by julien-deramond** [comment]
Could you please create a dedicated issue for that @AgentSmith0?

**Comment by saranglakare** [comment]
Any idea when this will be released? The problem exists on latest 5.3 as of 4 Feb 2023. Thanks!

**Comment by saranglakare** [comment]
@louismaximepiton @mdo could you please cross-check? The fix mentioned still works - so not sure if this change was never published or something caused this change to be reverted. Thanks a ton!

**Comment by louismaximepiton** [comment]
Hey Saranglakare, AgentQmith0,

Just pushed a PR fixing this issue, thanks for raising a flag here !

**Pull request merged**